arm-recoveryservices/2016-06-01/swagger/vaultusages.json

```diff
@@ -0,0 +1,188 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/usages": {
+      "get": {
+        "tags": [
+          "VaultUsages"
+        ],
+        "description": "Fetches the usages of the vault.",
+        "operationId": "VaultUsages_List",

```

Comment:
We are updating some of these guidelines. "Usages" should be the noun here, we plan to discourage such compound nouns moving forward. When fetching collections, the list operation should be named as either "CollectionItem_List" or "CollectionItem_ListByParent" where parent is the immediate preceding type in the url, which is "vaults" in this case